root/edit/components/EditSidebar.js

```diff
@@ -78,9 +79,13 @@ const EditSidebar = ({$c, edit}: Props) => (
     </SidebarProperties>
 
     <p>
-      <a href={`/edit/${edit.id}/data`}>
-        <bdi>{l('Raw edit data for this edit')}</bdi>
-      </a>
+      {$c.user_exists ? (
+        <a href={`/edit/${edit.id}/data`}>
+          <bdi>{l('Raw edit data for this edit')}</bdi>
+        </a>
+      ) : (
+        <RequestLogin $c={$c} text={l('Log in to see raw edit data')} />
```

Comment:
We probably shouldn't display anything in this case, since it's just technical data for developers. Anyone who needs this and isn't logged in knows what to do, while someone without an account isn't going to have any idea what this means or is for.